CHANGELOG.md

```diff
@@ -6,10 +6,15 @@ after the 3.9.0 release. All changes up until the 3.9.0 release can be found in
 The format is based on [Keep a Changelog](http://keepachangelog.com)
 
 ## [Unreleased]
+<!-- Keep this up to date! After a release, change the tag name to the latest release -->
+[Unreleased]: https://github.com/Adobe-Consulting-Services/acs-aem-commons/compare/acs-aem-commons-4.3.0...HEAD
 
 ### Fixed
 - #1975 - Split application content from mutable content
 
+### Added
+- #2017 - Added read/write access to `/conf` for acs-commons-on-deploy-scripts-service uwer
```

Comment:
Latest `master` needs to be merged into this ticket so that this can then be put back up in the "Unreleased" section.

Reply:
This comment in CHANGELOG still needs to be moved up to the "Unreleased" section.

Reply:
I thought it was handled already, done now.